### ISSUE TITLE
Cannot navigate to Training Setup Summary

### ISSUE THREAD

**Issue by beccamc** [dotnet/machinelearning-modelbuilder#959]
Using tab control to navigate through the Model Builder app.... 

Cannot access the Training setup summary button 

![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/117c595e-019e-4ac9-af82-1c3bb2a14610)

**Comment by beccamc** [comment]
This will be verified in the accessibility pass.
